src/FakeItEasy/A.cs

```diff
@@ -113,6 +113,7 @@ public static T Dummy<T>()
         /// <param name="typeOfDummy">The type of dummy to return.</param>
         /// <returns>A dummy object of the specified type.</returns>
         /// <exception cref="ArgumentException">Dummies of the specified type can not be created.</exception>
+        [EditorBrowsable(EditorBrowsableState.Advanced)]

```

Comment:
Sorry, I wasn't clear. I meant only the ones that are currently missing in master (there's only one).

Anyway, I'm happy as-is. No need to squash.